internal/commands/sbomtest/presenter.go

```diff
@@ -103,7 +97,7 @@ func resultToJSONOutput(res *snykclient.SBOMTestResult) JSONOutput {
 				CVSSv3:    vuln.CVSSv3,
 				CVSSScore: vuln.CVSSscore,
 
-				Severity:             severityWithoutCritical,
+				Severity:             vuln.SeverityLevel,
 				SeverityWithCritical: vuln.SeverityLevel,
```

Comment:
Is this a duplicated field?

Reply:
Yep, but it's expected, at least if we want to follow the managed output.

Reply:
Okay, suppose there can always be a follow up to refine outputs.